Makefile

```diff
@@ -240,7 +240,7 @@ tests/%.uplc.run: tests/%.uplc
 	$(CHECK) $<.out $<$(EXPECTED2)
 
 update-results: conformance-test
-update-results: TEST=$(UPLC) evaluate --print-mode Classic -i
+update-results: TEST=$(UPLC) evaluate --print-mode Classic -i 
```

Comment:
I don't think this whitespace addition was intended. Please remove the white space.